web/src/components/Navigation/Router.tsx

```diff
@@ -8,8 +8,8 @@ const Router = (): JSX.Element => {
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Home path="home" />} />
-        <Route path="/test/:id" element={<Test path="test-details" />} />
-        <Route path="/test/:testId/result/:resultId" element={<Trace path="trace" />} />
+        <Route path="/test/:testId" element={<Test path="test-details" />} />
+        <Route path="/test/:testId/result/:runId" element={<Trace path="trace" />} />
```

Comment:
Not important as part of this PR, but what do you think about renaming `result` to `run` in the path?

Reply:
Yes, makes sense